src/runtimes/node/zip_esbuild.js

```diff
@@ -9,6 +9,19 @@ const { zipNodeJs } = require('../../zip_node')
 const { bundleJsFile } = require('./bundler')
 const { getSrcFilesAndExternalModules } = require('./src_files')
 
+const getAliases = ({ bundlePath, mainFile, sourcemapPath, srcDir }) => {
```

Comment:
Extracted from `zipEsbuild` to conform with linter rules. The only functionality that changed here was that `aliases` is now a `Map` instead of the object, so that we can conditionally add elements to it.